deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -23,6 +23,9 @@ rollingUpdateMaxUnavailable: 1
 # Namespace to deploy agent in (Optional: Will default to release namespace)
 namespace:
 
+# Specify PSP (Pod Security Policy) (Optional: default to not using PSP mode if no values are provided by the user)
+pspMode: restricted/privileged
```

Comment:
This value is used as a default so it should be blank to indicate no security mode.